config/settings.yml

```diff
@@ -333,12 +342,17 @@ libraries:
       library_slug: sal12
       location_slug: operations
     pageable: true
+    aeon_sites:
+      L-PAGE-EA: EASTASIA
   SAL3:
     label: SAL3 (off-campus storage)
     hours:
       library_slug: sal3
       location_slug: operations
     pageable: true
+    aeon_sites:
```

Comment:
I wonder if the `aeon_site` configuration would be better on the `pageable` rule, so it's harder to the data out of sync?

Reply:
I like that; I think it'll result in less nesting in the YAML too. I'll try it